workspaces/ui/src/components/Page.js

```diff
@@ -34,6 +34,13 @@ export default function Page(props) {
     document.title = `${title} - Optic`;
   }, [title]);
 
+  const { pathname } = useLocation();
+  const scrollToTop = true; // change to be determined by page context
+
+  useEffect(() => {
+    window.scrollTo(0, 0);
+  }, [pathname && scrollToTop]);
+
```

Comment:
Great job, almost there! 🙂

- if `scrollToTop` comes from the `props` object, it can be used like `<Page scrollToTop={true}></Page>` in other components. There's [an excellent guide by the ReactJS team about props and how to use them](https://reactjs.org/docs/components-and-props.html), if you'd like to look that up.
- the `scrollToTop` must be used inside the `useEffect` to trigger the window scrolling conditionally. The second parameter `[pathname && scrollToTop]` is monitored to see whether the effect must be run. _Any change in that value_ causes the effect to be ran again. For more details, see [this guide to how useEffect works](https://reactjs.org/docs/hooks-overview.html#effect-hook).

Reply:
adjusted it! since the default behavior is not to scroll to the top, I also specified for `DocsPage` to pass in `scrollToTop`